src/components/Articles/CompanyProvisions.vue

```diff
@@ -8,19 +8,19 @@
             CHANGED
           </v-chip>
         </v-col>
-        <v-col cols="7" class="pt-0 pl-0 info-text" v-if="provisionsRemoved">
+        <v-col id="noneOfProvisionsApplyText" cols="7" class="pt-0 pl-0 info-text" v-if="provisionsRemoved">
```

Comment:
This is not critical, but this is not the ID format (style) of other components in our projects. Please use kebab-case for element IDs and classes. (We use snake-case for names that are used in the script, like refs or other properties or methods.)